deploy/eck-operator/templates/podMonitor.yaml

```diff
@@ -1,10 +1,13 @@
-{{- $metricsPort := int .Values.config.metricsPort -}}
+{{- $metricsPort := int (include "eck-operator.metrics.port" .)}}
 {{- if and .Values.podMonitor.enabled (gt $metricsPort 0) }}
```

Comment:
I've done the same here, although I thought a single fail in the configmap template would be sufficient, since you can't disable it's processing with a value, but it can do no harm.